mf4-csv.py

```diff
@@ -44,22 +46,12 @@
 }
 
 def main():
-    dotenv.load_dotenv()
-    conn = psycopg2.connect(
-        database="postgres", user=os.getenv("USER"), password=os.getenv("PASSWORD"), host=os.getenv("HOST"), port=os.getenv("PORT")   
-    )
-    cursor = conn.cursor()
+    dotenv.load_dotenv("./credentials.env")
+    file_convert(1)
 
-    data = cursor.execute("""
-    SELECT * from PVCSTATE
-    """)
-    print(data)
-    #file_convert()
 
-
-
-"""Converts .mf4 files to .csv files. There are 7 channel groups created in this conversion. We currently only use number 7.
-This fact will be hard coded into this process as I do not forsee a world where our Mech-E team ever uses another channel."""
+# Converts .mf4 files to .csv files. There are 7 channel groups created in this conversion. We currently only use number 7.
```

Comment:
Can you find a more standard way of commenting Python files? Indicate what standard you're following in the README